Disable the fallback on subprojects/gi-docgen -- if py-docgen is not found 
on the system, the shipped fallback in 'subprojects' would be tried. This is
unwanted behaviour.

As gtk_doc option is 'boolean' instead of 'feature', disabling the
option makes the dependency optional instead of disable the check.

https://gitlab.gnome.org/GNOME/pango/-/commit/cc9072ad21c2

--- meson.build.orig	2021-05-27 18:22:12 UTC
+++ meson.build
@@ -585,7 +585,6 @@ pango_conf.set('HAVE_SYSPROF', libsysprof_capture_dep.
 pango_deps += libsysprof_capture_dep
 
 gidocgen_dep = dependency('gi-docgen', version: '>= 2021.1',
-                          fallback: ['gi-docgen', 'dummy_dep'],
                           required: get_option('gtk_doc'))
 
 gnome = import('gnome')
